images/base/containerd-config.toml

```diff
@@ -2,10 +2,15 @@
 disabled_plugins = ["aufs", "btrfs", "zfs"]
 
 # set default runtime handler to v2, which has a per-pod shim
-[plugins.cri.containerd.default_runtime]
-  runtime_type="io.containerd.runc.v2"
+[plugins."io.containerd.grpc.v1.cri".containerd]
+  default_runtime_name="runc"
+[plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc]
+  runtime_type = "io.containerd.runc.v2"
```

Comment:
xref: https://github.com/containerd/cri/blob/815f3e727de84cda4c74ac4324d6469d20df94f7/cluster/gce/configure.sh#L180-L183

Reply:
:thinking: in the long term, how can we be sure we don't deviate from the official cri?

Reply:
huh?

Reply:
you are getting the configs models from a bash script in the containerd/cri repo, I was wondering how we can track any changes there, it can happen that they change things and we deviate, thus we'll be testing different things.

Reply:
No, this is documented elsewhere. I'm just referencing that in this regard we're the same as another user, we have no reason not to be.

Reply:
This is both in the release notes and in the config docs upstream. We're just opting into the v2 (pod shim)